Sources/MapboxMaps/Gestures/GestureManager+GestureHandlerDelegate.swift

```diff
@@ -0,0 +1,186 @@
+import UIKit
+import MapboxCommon
+import MapboxCoreMaps
+
+extension GestureManager: GestureHandlerDelegate {
+    // MapView has been tapped a certain number of times
+    internal func tapped(numberOfTaps: Int, numberOfTouches: Int) {
+
+        guard let mapView = cameraManager.mapView else {
+            return
+        }
+
+        // Single tapping twice with one finger will cause the map to zoom in
+        if numberOfTaps == 2 && numberOfTouches == 1 {
+            cameraManager.setCamera(to: CameraOptions(zoom: mapView.zoom + 1.0),
+                                    animated: true,
+                                    duration: 0.3,
+                                    completion: nil)
+        }
+
+        // Double tapping twice with two fingers will cause the map to zoom out
+        if numberOfTaps == 2 && numberOfTouches == 2 {
+            cameraManager.setCamera(to: CameraOptions(zoom: mapView.zoom - 1.0),
+                                    animated: true,
+                                    duration: 0.3,
+                                    completion: nil)
+        }
+    }
+
+    internal func panBegan(at point: CGPoint) {
+        try! cameraManager.mapView?.__map.dragStart(forPoint: point.screenCoordinate)
```

Comment:
I believe this is the use of the drag API. I'm concerned that we are not handling the force try. I'm new to Drag API so I think we should really catch this error here.

Reply:
This isn't actually a throwing function -- this is a result of the fact that the drag API exists on the map  object. All APIs on the `map` are currently marked as `throws`. This will be changing in the near future. We'll handle this case along with the other instances of `try!`.

Reply:
gotcha thanks for the clarification